programs/libraries/rjson.scl

```diff
@@ -0,0 +1,109 @@
+//raw-json string to text
+if(0,    //if you are using an old version, change it to 1
+rjson(rj)->(
+barname='chyxrjson:rjson'+(rand(0.9)-'.');
+if(bossbar(barname)==false,return(rjson(rj)));
+command=run('bossbar set '+barname+' name '+rj);
+res=bossbar(barname,'name');
+bossbar(barname,'remove');
+if (command:1,res,throw('not valid r-json'))
+);
+
+
+,
+rjson(rj)->(

```

Comment:
can you put comments here describing what this function is meant to do?

Reply:
ok